types/stripe/stripe-tests.ts

```diff
@@ -282,6 +282,11 @@ stripe.checkout.sessions.create({
     // asynchronously called
 });
 
+stripe.checkout.sessions.retrieve('ch_test_123');
+stripe.checkout.sessions.retrieve('ch_test_123', { expand: ['payment_intent'] }).then(session => {
+    session && typeof session.payment_intent !== 'string' && session.payment_intent.id;
```

Comment:
These tests are only checked for types, not actually invoked. You'll want to do something like this:

```ts
session.payment_intent; // $ExpectType PaymentIntent | string
```